trade-finance-manager-ui/server/controllers/case/amendments/amendmentFacilityEndDate.get.api-test.js

```diff
@@ -199,6 +199,108 @@ describe('amendmentFacilityEndDate routes', () => {
           user,
         });
       });
+
```

Comment:
Added 3 tests in this file, and essentially the exact same 3 tests in the `post` test file below. (diff is harder to parse in the other file as I've also wrapped two sections of tests in `describe` blocks)